web/less/chat.less

```diff
@@ -36,142 +25,210 @@
 
-        .time {
-          color: #c5bbb4;
+        p {
+          background: none;
+          margin-right: 0;
         }
+      }
 
-        .sender {
-          color: #282828;
-          padding-left: 10px;
-        }
+      li.yourself {
+        align-self: flex-end;
 
         & > p {
-          color: #151515;
-          font-size: 11px;
-          word-break: keep-all;
-          margin-top: 2px;
-          margin-bottom: 2px;
-          padding: 5px;
-          box-sizing: border-box;
-
-          p {
-            margin-top: 2px;
-            margin-bottom: 2px;
-          }
-        }
-
-        & > p.yourself {
+          border-right: 2px solid #10A9DC;
           text-align: right;
-          background-color: rgba(220, 220, 220, .4);
           margin-left: 40px;
-          width: calc(~"100% - 40px");
-          border-radius: 4px;
+          margin-right: 0;
 
           & > p {
             text-align: left;
           }
         }
 
-        a {
-          background: none;
-          display: inline;
-          text-decoration: underline;
-          margin: -12px -15px;
-          padding: 12px 15px;
+      }
 
-          &:link {
-            color: #1a99c3;
+      li {
+          align-self: flex-start;
+          background-color: transparent;
+          padding: 10px 30px 10px 20px;
+          border: 0;
+          color: #3f4f57;
+          width: fit-content;
+
+          .time, .sender {
+            font-size: 12px;
           }
 
-          &:hover,
-          &:active {
-            color: #0d4c62;
+          .sender {
+            color: #282828;
+            padding-left: 10px;
           }
 
-          &:visited {
-            color: #76c2db;
+          & > p {
+            color: #151515;
+            background-color: #ffffff;
+            font-size: 16px;
+            word-break: break-all;
+            margin-right: 40px;
+            box-sizing: border-box;
+            padding-left: 20px;
+            padding-right: 20px;
+            padding-top: 16px;
+            padding-bottom: 16px;
+            margin-top: 0;
+            margin-bottom: 0;
+
+            p {
+              margin-top: 5px;
+              margin-bottom: 0;
+            }
+          }
+
+          a {
+            background: none;
+            display: inline;
+            text-decoration: underline;
+            margin: -12px -15px;
+            padding: 12px 15px;
+
+            &:link {
+              color: #1a99c3;
+            }
+
+            &:hover,
+            &:active {
+              color: #0d4c62;
+            }
+
+            &:visited {
+              color: #76c2db;
+            }
           }
-        }
 
-        a, p, span {
-          user-select: text;
+          a, p, span {
+            user-select: text;
+          }
+
+          // Loop to make different color borders for chat users.
+          @chat-colors:  #DC0CE8, #54CC14, #F5A623,#C41F4B, #E8520C, #600DFF, #54CC14, #54CC14, #54CC14;
```

Comment:
This list includes the color `#54CC14` four times. Also, does this work when there are more than 10 participants chatting? (Do you use the `%` operator in the JavaScript that assigns the counter number?)

Reply:
@jeffswartz Sorry I was waiting for the color list from jmo. I'll add it now.
As for the operator I've been converting to a string and taking the last digit.
```
 // We only have 10 colors so just get last digit.
 +      var participantNumber = chatIndex.toString().slice(-1);
 +      info.data('participant-number', participantNumber);
```
That code section is here: https://github.com/opentok/OpenTokRTC-V2/pull/379/files#diff-3da587dcf2ac43d5ed53ab96b1524bf7R198